apps/docs/src/scenes/DOMParticles.tsx

```diff
@@ -114,11 +114,17 @@ export const DOMParticles = () => {
         Math.random() * s - s * 0.5;
     const randomVector = (s: number) =>
         new Vector2(rnd(s), rnd(s));
-    const rndPos = () =>
-        randomVector(offset).add(new Vector2(
-            containerRef.current.offsetWidth / 2,
-            containerRef.current.offsetHeight / 2,
-        ))
+    const rndPos = () => {
+        const container = containerRef.current;
```

Comment:
Here's an example of a bug I fixed (but there are more like it). It errors during the render function when the container unmounts. I think it would get swallowed in production, but on the dev server it shows up as an error blocking the screen that you have to X out